packages/block-editor/src/components/link-control/test/index.js

```diff
@@ -1808,9 +1609,7 @@ describe( 'Addition Settings UI', () => {
 			);
 		};
 
-		act( () => {
-			render( <LinkControlConsumer />, container );
-		} );
+		const { container } = render( <LinkControlConsumer /> );
 
 		// Grab the elements using user perceivable DOM queries.
 		const settingsLegend = Array.from(
```

Comment:
The fieldset query is replaceable with:

```jsx
		const settingsFieldset = screen.getByRole( 'group', {
			name: 'Currently selected link settings',
		} );
```

Reply:
Updated in 3fda6b6645661348e6b88a1cbe817a518b8fb295, though made obsolete by b31e97f3000b13236467e24898ebbe30fae28274